tfgcv/testdata/full_container_cluster.tf

```diff
@@ -0,0 +1,133 @@
+resource "google_container_cluster" "full_list_default_1" {
+  # TODO: test beta fields.
+  # addons_config.cloudrun_config
+  # addons_config.cloudrun_config
+  # addons_config.cloudrun_config
+  # addons_config.istio_config
+  # authenticator_groups_config
+  # cluster_autoscaling
+  # database_encryption
+  # enable_binary_authorization
+  # enable_intranode_visibility
+  # enable_shielded_nodes
+  # enable_tpu
+  # node_config.workload_metadata_config
+  # node_config.sandbox_config
+  # pod_security_policy_config
+  # release_channel
+  # resource_usage_export_config
+  # vertical_pod_autoscaling
+  # workload_identity_config
+
+  # TODO: need a seperate test case for fields in conflict with others.
+  # ip_allocation_policy.cluster_ipv4_cidr_block
+  # ip_allocation_policy.create_subnetwork
+  # ip_allocation_policy.node_ipv4_cidr_block
+  # ip_allocation_policy.services_ipv4_cidr_block
+  # ip_allocation_policy.subnetwork_name
+  # region
+  # zone
+
+  # TODO: problematic fields
+  # node_pool
+
+  name = "test-cluster"
+
+  additional_zones = ["test-additional_zones"]
```

Comment:
Not sure if we should be putting invalid values in here. At the very least, should probably be real zones.